tests/test_data/test_pipelines/test_transform/utils.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+
+from mmdet.core.mask import BitmapMasks, PolygonMasks
+
+
+def _check_fields(results, pipeline_results, keys):
+    for key in keys:
+        if isinstance(results[key], (BitmapMasks, PolygonMasks)):
+            assert np.equal(results[key].to_ndarray(),
+                            pipeline_results[key].to_ndarray()).all()
+        else:
+            assert np.equal(results[key], pipeline_results[key]).all()
+            assert results[key].dtype == pipeline_results[key].dtype
+
+
+def check_result_same(results, pipeline_results):
```

Comment:
Please add docstring for this function.
The function name cannot accurately make people understand its function.